formatter/src/formatter/mac.rs

```diff
@@ -162,6 +162,13 @@ pub fn format_macro(
                 source,
                 mac.mac.tokens.clone(),
             );
+            let dos_endings = source
+                .raw_lines()
+                .nth(0)
+                .unwrap()
+                .to_string()
+                .ends_with("\r\n");
+            printer.set_dos(dos_endings);
```

Comment:
```suggestion
             let mut printer = Printer::new(PrinterSettings { ..settings.into(), crlf_line_endings });
```

Reply:
`crlf_line_endings` moved from `Printer` to `PrinterSettings` struct as per suggestion